tests_adapters/test_adapter_embeddings.py

```diff
@@ -21,34 +21,34 @@ def test_load_embeddings(self):
 
         self.assertEqual(model.active_embeddings, "test")
 
-    def test_add_embeddings(self):
+    def test_add_embeddings(self, embedding_dim=None):
         model = self.get_model()
         tokenizer = AutoTokenizer.from_pretrained("tests_adapters/fixtures/SiBERT")
-        model.add_embeddings("test", tokenizer)
+        model.add_embeddings("test", tokenizer, embedding_dim=embedding_dim)
```

Comment:
Here and other places with this change: Shouldn't we rather try to figure out `embedding_dim` automatically in `add_embeddings()` as `embedding_size` is part of the model config?